introduction.md

```diff
@@ -1,3 +1,5 @@
 reviewing it, so when got asked I still remember.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?